draft-ietf-dnsop-svcb-https.md

```diff
@@ -131,9 +131,9 @@ See {{?I-D.tapril-ns2}} as one example.
 The goal of the SVCB RR is to allow clients to resolve a single
 additional DNS RR in a way that:
 
-* Provides service endpoints authoritative for the service,
+* Provides alternative endpoints that are authoritative for the service,
```

Comment:
While this doesn't need to be addressed before merging, I'm not certain that the concept of "authoritative" is the same in all possible protocols, particularly ones where the scope of a "service" might vary.  We might want to consider whether that's a term we want to keep as well.